test/ectest.c

```diff
@@ -1083,6 +1083,46 @@ static int char2_field_tests(void)
     BN_free(yplusone);
     return r;
 }
+
+static int hybrid_point_encoding_test(void)
```

Comment:
Shouldnt there be a negative test also, for a bad encoding?

Reply:
I have tweaked the test so it exercises the newly added y_bit check.

Reply:
Thanks.